test/unit/assignments.spec.ts

```diff
@@ -6,6 +6,23 @@ const fs = require("fs");
 
 export class AssignmentTests {
```

Comment:
Does this mean we can get rid of the assignments translation test file? Or is it already gone on this branch?

Reply:
Not in this branch, but it's removed in master, so when things get merged it should be gone.